test/utils/validation/PenaltyDetailsValidator.test.ts

```diff
@@ -0,0 +1,183 @@
+import ApiClient from 'ch-sdk-node/dist/client';
+import { LateFilingPenaltyService, Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { assert, expect } from 'chai';
+import { Request } from 'express';
+import { createSubstituteOf } from '../../SubstituteFactory';
+import { createSession } from '../session/SessionFactory';
+
+import { SESSION_NOT_FOUND_ERROR, TOKEN_MISSING_ERROR } from 'app/controllers/processors/errors/Errors';
+import { PenaltyDetailsValidator } from 'app/controllers/validators/PenaltyDetailsValidator';
+import { AuthMethod, CompaniesHouseSDK } from 'app/modules/Types';
+
+describe('PenaltyDetailsValidator', () => {
+    const createSDK = (apiResponse: any): CompaniesHouseSDK => {
+        const lateFillingPenaltyService = createSubstituteOf<LateFilingPenaltyService>(config => {
+            config.getPenalties(companyNumber).resolves(apiResponse);
+        });
+        const chApi: ApiClient = createSubstituteOf<ApiClient>(config => {
+            config.lateFilingPenalties.returns!(lateFillingPenaltyService);
+        });
+        return (_: AuthMethod) => chApi;
+    };
+    const companyNumber = 'NI000000';
+    const getRequest = (penaltyReference: string): Request => {
+        return {
+            body: {
+                companyNumber,
+                penaltyReference
+            },
+            session: createSession('secret', true)
+        } as Request;
+    };
```

Comment:
Woops, I should have been the one writing these tests. Sorry for that.

I'd suggest empty lines between each test